src/main/webapp/app/exam/manage/students/exam-students.component.ts

```diff
@@ -233,4 +232,18 @@ export class ExamStudentsComponent implements OnInit, OnDestroy {
         this.jhiAlertService.error(error);
         this.isTransitioning = false;
     }
+
+    /**
+     * Registers all students who are enrolled in the course for the exam
+     */
+    registerAllStudentsFromCourse() {
+        if (this.exam && this.exam.id) {
```

Comment:
is it possible for an exam to have the id 0? Not right?
```suggestion
        if (this.exam?.id) {
```

Reply:
good catch!